Libraries/TestReport.psm1

```diff
@@ -295,10 +295,13 @@ Class TestSummary
 		if ( $global:BaseOSVHD ) {
 			$str += "`r`nVHD Under Test        : " + $global:BaseOSVHD
 		}
-		if ( $global:ARMImageName ) {
+		if ($global:ARMImageName) {
 			$imageInfo = $global:ARMImageName.Split(' ')
 			$str += "`r`nARM Image Under Test  : " + "$($imageInfo[0]) : $($imageInfo[1]) : $($imageInfo[2]) : $($imageInfo[3])"
 		}
+		if ($global:OverrideVMSize) {
```

Comment:
if the override size existed in the test case definition, it no need to show in test summary, right?

Reply:
I disagree. The intention here is for one stop shopping in the LISAv2 summary page. Both cases (pre-defined in xml and custom parameter in the run-time) will display VM size.